source/common/upstream/upstream_impl.cc

```diff
@@ -63,13 +63,21 @@ namespace {
 
 AddressSelectFn
 getSourceAddressFnFromBindConfig(const envoy::config::core::v3::BindConfig& bind_config) {
+  if (bind_config.additional_source_addresses_size() > 1) {
+    throw EnvoyException("Only one additional address can be supported");
```

Comment:
It's unclear what field this applies to. Can you make this message more clear and potentially also include the name of the cluster it applies to? Same below.

Reply:
got it, let me update